CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
 
+### Added
+- Add the `etcd-advertise-client-urls` config attribute to sensu-backend
```

Comment:
My ocd is suggesting you add a newline here.

Reply:
Fixed via https://github.com/sensu/sensu-go/pull/2375/commits/e51fbaebda392863ddbf56543e9d16659c134e28